hack/install.sh

```diff
@@ -11,13 +11,57 @@
 VERSION=${DAYTONA_SERVER_VERSION:-"latest"}
 BASE_URL=${DAYTONA_SERVER_DOWNLOAD_URL:-"https://download.daytona.io/daytona"}
 DESTINATION=${DAYTONA_PATH:-"/usr/local/bin"}
+CONFIRM_FLAG=false
+
+# Check for the -y flag
+for arg in "$@"; do
+  case $arg in
+    -y)
+    CONFIRM_FLAG=true
+    shift
+    ;;
+  esac
+done
 
 # Print error message to stderr and exit
 err() {
   echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
   exit 1
 }
 
+# Stop the Daytona server if it is running
+stop_daytona_server() {
+  if pgrep -x "daytona" > /dev/null; then
+    if [ "$CONFIRM_FLAG" = false ]; then
+      read -p "Daytona server is running. Do you want to stop it? (yes/no): " user_input
```

Comment:
@ezhil56x you can emulate interactive shell when running script in non-interactive by explicitly open /dev/tty for input like
```
      read -p "Daytona server is running. Do you want to stop it? (yes/no): " user_input < /dev/tty
```

Reply:
Cool it works! Updated my PR